README.md

```diff
@@ -33,9 +33,10 @@ These videos on TikTok, YouTube and Instagram get MILLIONS of views across all p
 1. Clone this repository
 2. Rename `.env.template` to `.env` and replace all values with the appropriate fields. To get Reddit keys (**required**), visit [the Reddit Apps page.](https://www.reddit.com/prefs/apps) TL;DR set up an app that is a "script". Copy your keys into the `.env` files.
 3. Run `pip3 install -r requirements.txt`
-4. Run `python3 main.py`
-5. ...
-6. Enjoy 😎
+4. Run `npx playwright install`
```

Comment:
why use `npx` here? did you try `python3 -m playwright install`?

Reply:
@SpaYco Does doing that work for you? This arguably would be a better solution.

Reply:
I agree with the approach @SpaYco suggested. Lot's of people won't have npm installed to use npx.